addons/account/views/account_move_views.xml

```diff
@@ -153,6 +153,7 @@
                     <field name="debit" sum="Total Debit"/>
                     <field name="credit" sum="Total Credit"/>
                     <field name="amount_currency" groups="base.group_multi_currency" optional="hide"/>
+                    <field name="currency_symbol" groups="base.group_multi_currency" optional="hide"/>
```

Comment:
is it really worth adding a new field for the currency symbol, where we could have simply put an optional=hide on the existing currency_id field

@tsb-odoo ?

Reply:
It's probably my fault, as I thought the symbol was wanted. But if the currency name is enough, we could even make the change in the earlier versions.

Reply:
@qdp-odoo there might be confusion ; the ultiimate goal is that while exporting the CSV, the user knows which currency is used for each and every line of the partner ledger. No matter if it's a symbol (€ or $) or rather EUR / USD. Up to now, amounts in different currencies are exported as flat amounts, resulting in a CSV where currency info is lost. EUR can then be summed with USD... > not correct.

Reply:
> It's probably my fault, as I thought the symbol was wanted. But if the currency name is enough, we could even make the change in the earlier versions.

@qdp-odoo and @vin-odoo ok then. To me, currency name is enough.